modules/service/variables.tf

```diff
@@ -83,3 +83,20 @@ variable "tags" {
   type        = map(string)
   default     = {}
 }
+
+variable "service_registry_arn" {
+  default     = ""
+  description = "ARN of aws_service_discovery_service resource"
+  type        = string
+}
+
+variable "container_name" {
+  description = "Optional name for the container to be used instead of name_prefix. Useful when when constructing an imagedefinitons.json file for continuous deployment using Codepipeline."
+  default     = ""
+  type        = string
+}
+
+variable "task_container_port" {
```

Comment:
Tests fail due to this now being required, we need to have a default value or this will be a breaking change.

Reply:
I think this variable is not necessary since port is specified in the `target` variable? Ref:
https://github.com/telia-oss/terraform-aws-ecs/blob/master/modules/service/main.tf#L17

Reply:
Agree, done.